.github/workflows/reusable_pull_request.yml

```diff
@@ -159,3 +167,27 @@ jobs:
       env:
         PROJECT_NAME: ${{ inputs.license_header_check_project_name }}
       run: curl -s https://raw.githubusercontent.com/apple/swift-nio/main/scripts/check-license-header.sh | bash
+
+  broken-symlink-check:
+    name: Broken symlinks check
+    if: ${{ inputs.broken_symlink_check_enabled }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+    - name: Run broken symlinks check
+      run: curl -s https://raw.githubusercontent.com/apple/swift-nio/main/scripts/check-broken-symlinks.sh | bash
+
+  formatk-check:
+    name: Format check
+    if: ${{ inputs.format_check_enabled }}
+    runs-on: ubuntu-latest
+    container:
+      image: swiftlang/swift:nightly-6.0-jammy
+    timeout-minutes: 5
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+    - name: Run format check
+      run: swift format format --parallel --recursive --in-place
```

Comment:
is 'format format' expected?

Reply:
Yes it is. `swift-format` now ships with the toolchain and you can reach it by calling `swift format`. The second `format` is the argument to trigger the formatting action. Now that you say this we should probably call `swift format lint` though